service/models.py

```diff
@@ -13,31 +14,43 @@
 
 
 class DataValidationError(Exception):
-    """ Used for an data validation errors when deserializing """
+    """Used for an data validation errors when deserializing"""
 
 
-class YourResourceModel(db.Model):
+class PaymentType(Enum):
+    """Enumeration of valid payment types"""
+
+    UNKNOWN = 0
+    CREDITCARD = 1
+    PAYPAL = 2
+
+
+class PaymentMethod(db.Model):
     """
-    Class that represents a YourResourceModel
+    Class that represents a PaymentMethod
     """
 
     ##################################################
     # Table Schema
     ##################################################
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
-
-    # Todo: Place the rest of your schema here...
+    name = db.Column(db.String(63), nullable=False)
+    payment_type = db.Column(
+        db.Enum(PaymentType), nullable=False, server_default=(PaymentType.UNKNOWN.name)
+    )
+    payment_type_id = db.Column(db.Integer)
```

Comment:
I think this should also be `non-nullable`?

Reply:
Do we want to create the entry in the PaymentMethod table first or the 'PaymentType' table first? 

Since the entries are supposed to be pointing to each other, one of them would have to be nullable first and updated later when the other entry is created (I think).